commit 4dc8b2e57ee4f8c8ab2fcdadb927e023ac1a1d7e
Author: Geert Uytterhoeven <geert+renesas@glider.be>
Date:   Thu Jan 10 14:39:15 2019 +0100

    arm64: dts: renesas: r8a7796: Enable DMA for SCIF2
    
    [ Upstream commit 97f26702bc95b5c3a72671d5c6675e4d6ee0a2f4 ]
    
    SCIF2 on R-Car M3-W can be used with both DMAC1 and DMAC2.
    
    Fixes: dbcae5ea4bd27409 ("arm64: dts: r8a7796: Enable SCIF DMA")
    Signed-off-by: Geert Uytterhoeven <geert+renesas@glider.be>
    Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
    Signed-off-by: Sasha Levin <sashal@kernel.org>

diff --git a/arch/arm64/boot/dts/renesas/r8a7796.dtsi b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
index cbd35c00b4af..33cb0281c39c 100644
--- a/arch/arm64/boot/dts/renesas/r8a7796.dtsi
+++ b/arch/arm64/boot/dts/renesas/r8a7796.dtsi
@@ -1161,6 +1161,9 @@
 				 <&cpg CPG_CORE R8A7796_CLK_S3D1>,
 				 <&scif_clk>;
 			clock-names = "fck", "brg_int", "scif_clk";
+			dmas = <&dmac1 0x13>, <&dmac1 0x12>,
+			       <&dmac2 0x13>, <&dmac2 0x12>;
+			dma-names = "tx", "rx", "tx", "rx";
 			power-domains = <&sysc R8A7796_PD_ALWAYS_ON>;
 			resets = <&cpg 310>;
 			status = "disabled";
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1068_4dc8b2e57ee4f8c8ab2fcdadb927e023ac1a1d7e.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1068_4dc8b2e57ee4f8c8ab2fcdadb927e023ac1a1d7e.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
